Automattic/jetpack
# PHPCS: Comment Likes Module

This PR updates the comment likes module to meet phpcs standards.

#### Changes proposed in this Pull Request:

* This PR updates the comment likes module to meet phpcs standards.

#### Jetpack product discussion



N/a

#### Does this pull request change what data or activity we track or use?



Nope.

#### Testing instructions:






* Check out branch
* Make sure you can still enable comment likes under Discussion > Comments.
* Check out comments, make sure you can see likes, how many there are, view the like table in the comments dashboard.
